cvat-ui/src/components/signing-common/social-account-card.tsx

```diff
@@ -0,0 +1,40 @@
+// Copyright (C) 2023 CVAT.ai Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import './styles.scss';
+import React from 'react';
+import Button from 'antd/lib/button/button';
+import Tooltip from 'antd/lib/tooltip';
+import { SocialAccountLinkProps } from './social-account-link';
+
+function SocialAccountCard(props: SocialAccountLinkProps): JSX.Element {
+    const svgWrapperRef = React.useRef();
+    const {
+        children, className, href, icon,
+    } = props;
+
+    React.useEffect(() => {
+        if (icon) {
+            // eslint-disable-next-line no-unsanitized/property
+            svgWrapperRef.current.innerHTML = icon;
+        }
+    }, [icon, svgWrapperRef.current]);
```

Comment:
Is icon just a string-encoded svg image? If so, I think its better to show it like on [model's form](https://github.com/opencv/cvat/blob/develop/cvat-ui/src/components/models-page/model-provider-icon.tsx). We dont really need a ref - setting innerHtml.